src/integrator_interface.jl

```diff
@@ -9,6 +9,9 @@ function savevalues!(integrator::DDEIntegrator, force_save=false)
     integrator.integrator.k = integrator.k
     integrator.integrator.t = integrator.t
 
+    # update solution of DDE integrator
+    invoke(savevalues!, Tuple{AbstractDDEIntegrator,Bool}, integrator, force_save)
```

Comment:
No, these are two different solutions. One reason is that often other components of the history function have to be evaluated as well, e.g. if the system would be ``x'(t) = y(t-1), y'(t) = x(t-2) + y(t)`` then one can not evaluate ``x(t-2)`` if one just restricts the solution of the ODE integrator to ``y`` (as currently done).